### PR TITLE
Move IServiceCollection extension to Microsoft.Framework.DependencyInjec...

### PR DESCRIPTION
**Pull request by Praburaj**
...tion namespace

Fixes: https://github.com/aspnet/Diagnostics/issues/96

@muratg

### CODE REVIEW

File: src/Microsoft.AspNet.Diagnostics.Elm/ElmServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Jetbrains.Annotations;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.Diagnostics.Elm
```
Comment (Praburaj) [inline src/Microsoft.AspNet.Diagnostics.Elm/ElmServiceCollectionExtensions.cs]: Oops. Fixing the namespace :smile:

File: src/Microsoft.AspNet.Diagnostics.Elm/ElmServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+using Microsoft.AspNet.Diagnostics.Elm;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class ElmServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Registers an <see cref="ElmStore"/> and configures <see cref="ElmOptions"/>.
+        /// </summary>
+        public static IServiceCollection AddElm([NotNull] this IServiceCollection services, Action<ElmOptions> configureOptions = null)
```
Comment (davidfowl) [inline src/Microsoft.AspNet.Diagnostics.Elm/ElmServiceCollectionExtensions.cs]: No more options params.

Comment (Praburaj) [inline src/Microsoft.AspNet.Diagnostics.Elm/ElmServiceCollectionExtensions.cs]: you mean optional params?

**Comment by Praburaj** [comment]
Updated removing optional parameters.

File: src/Microsoft.AspNet.Diagnostics.Elm/ElmServiceCollectionExtensions.cs (hunk 2 above)
Comment (muratg) [inline src/Microsoft.AspNet.Diagnostics.Elm/ElmServiceCollectionExtensions.cs]: Yep.  Default values are ok only for CancellationTokens. Remove ```= null``` from the end and pass it explicitly where needed.

**Comment by muratg** [comment]
Looks good, unless @davidfowl has more comments, :shipit:

**Pull request merged**